package.json

```diff
@@ -22,25 +22,28 @@
     "prebuild": "rm -fr dist/* && npm run gen-data-json",
     "version": "node scripts/set-version.js",
     "build": "tsc && node build.mjs",
-    "build:image": "npm run build && docker buildx build --tag pepr:dev .",
+    "build:image": "npm run build && docker buildx build --output type=docker --tag pepr:dev .",
```

Comment:
this `--output type=docker` is something I have to do locally but not necessary for the PR